blog/2023-03-01-recommended-architecture-for-a-simple-payment-dapp.md

```diff
@@ -0,0 +1,107 @@
+# Recommended architecture for a simple payment dapp
```

Comment:
Can we call this "Example" since we are not really recommending to build in vanilla